review-tools/README

```diff
@@ -101,16 +105,37 @@ Checkout branch for query 145:
 ghmerge
 -------
 
-ghmerge merges (reviewed and approved!) GitHub pull requests
-
-It works on the current branch, which should be master or one of the stable
-releases.
-
-Usage:
-        ghmerge ### reviewer...
-
-The ### is the GitHub MR number.  The rest of the args are the names of
-the reviewers (passed to addrev; see above).
+ghmerge calls addrev and pushes (reviewed and approved!) GitHub pull requests.
+It includes several safety precautions and questions such as showing the diff,
+showing the resulting commit messages, and (by default) rebuilding everything.
+
+It works on the current branch, which should be 'master' or one of the stable
+releases. For the branch to be merged a suitable remote must be available.
+
+The tool accesses external databases via HTTPS (e.g., on api.github.com).
+The environment variables 'https_proxy' and 'no_proxy' can be used.
+
+Example usage patterns:
+        ghmerge <prnum> <reviewers>...
+        ghmerge --tools --nomerge <prnum> <reviewer>...
+
+Available options are:
+
+--nomerge         Use 'git rebase -i --autosquash'.
+                  This is the default.
+--autosquash      An alias to '--nomerge'.
+--merge           Use 'git merge --no-commit --squash'.
+--squash          An alias to '--merge'.
+--nobuild         Do not use 'opensslbuild'.
+                  Else it is invoked with $CC defaulting to "ccache gcc".
+--tools           Select the 'tools' repository rather than 'openssl'.
+                  This implies '--merge' and '--nobuild'.
```

Comment:
I would rather have `--autosquash` the default *everywhere*, at the very least for consistency.

Reply:
Agreed.  With almost everyone using fixup commits, it makes sense.

Reply:
Since `--autosquash` is, as written in the above addition to the README, an alias to `--nomerge` and `--nomerge` has recently become the default according to this PR, `--autosqash` is already the default.
Note also that for the alternative approach (triggered by `--merge` or  `--squash`) using `git merge --squash`, the `--autosquash` option is not supported by git.

Reply:
> `--autosqash` is already the default.

Then please explain to me the `MERGE=yes` in the lines quoted below.

``` shell
    --tools)
        WHAT=tools ; MERGE=yes ; BUILD=no ; shift
        ;;
    --web)
        WHAT=web ; MERGE=yes ; BUILD=no ; shift
        ;;
```

Reply:
Ah, now I understand what you meant. Right, whoever did those  `--tools` and `--web` options for some (strange) reason added `MERGE=yes` to them. 

I was already considering dropping `MERGE=yes` for them. Done now.